securedrop/source_templates/tor2web-warning.html

```diff
@@ -1,4 +1,7 @@
 {% extends "base.html" %}
+
+{% block tab_title %}WARNING: {{ gettext('Proxy Service Detected') }}{% endblock %}
```

Comment:
While we do need all user-facing strings to be [wrapped in `gettext()`] for translation, rest assured that you are not responsible for supplying or managing it!  Our release process involves a localization phase.  :-)

```suggestion
{% block tab_title %}{{ gettext('WARNING: Proxy Service Detected') }}{% endblock %}
```

[wrapped in `gettext()`]: https://developers.securedrop.org/en/latest/i18n.html

Reply:
Fixed. I also added gettext calls around other title blocks which were all plain text, let me know if that was incorrect. I uploaded this as a separate commit so that it was easier for you to see the updates, but I can squash everything before you accept the pull request so that the history of develop stays clean.

Reply:
Thanks, @saffronsnail!